tests/semantic_analyser.cpp

```diff
@@ -1460,6 +1460,11 @@ TEST(semantic_analyser, tracepoint)
   test("tracepoint:category:event { 1 }", 0);
 }
 
+TEST(semantic_analyser, rawtracepoint)
+{
+  test("rawtracepoint:event { 1 }", 0);
```

Comment:
Can we add at least one negative test-case?
```suggestion
  test("rawtracepoint:event { 1 }", 0);
  test("rawtracepoint:category:event { 1 }", 1);
```